src/pages/transactions/details.tsx

```diff
@@ -152,6 +160,46 @@ const TransactionDetails = () => {
                 {tx.contract.num_msgs > 1 && (
                   <Text type="h6" color="white" margin="0px 0px 12px">{`#${i + 1}`}</Text>
                 )}
+                {(v?.grc20 === true || v?.pkg_path !== '/bank.MsgSend') && (
+                  <>
+                    <DLWrap desktop={desktop}>
+                      <dt>Name</dt>
+                      <dd>
+                        <Badge>
+                          <Text type="p4" color="white">
+                            {v?.pkg_name || '-'}
+                          </Text>
+                        </Badge>
+                      </dd>
+                    </DLWrap>
+                    <DLWrap desktop={desktop}>
+                      <dt>
+                        Path
+                        <div className="tooltip-wrapper">
+                          <Tooltip content={TOOLTIP_PACKAGE_PATH}>
+                            <IconTooltip />
+                          </Tooltip>
+                        </div>
+                      </dt>
+                      <dd>
+                        <Badge>
+                          <Text type="p4" color="blue" className="ellipsis">
+                            <Link href={`/realms/details?path=${v.pkg_path}`} passHref>
```

Comment:
if user called tx with package(path) that doesn't exist on chain,
this link will always fail (or return 404)